lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -373,12 +374,6 @@ func (p *ConnPool) Create(table, key string, data interface{}) *errors.Error {
 */
 func (p *ConnPool) Update(table, key string, data interface{}) (string, *errors.Error) {
 
```

Comment:
previously it was informed this code has already gone in. Why is this getting merged now